### DISCUSSION TITLE
The result of build is empty

### DISCUSSION THREAD

**Discussion by xixiaby** [allure-framework/allure2#1714] [Questions & Support] [answered]
After I  bulid the project ,the result "allure2-2.19-SNAPSHOT.jar" under directory "build/lib" is empty. 
What should I do?

**Comment by baev** [comment, marked as answer]
* `allure-commandline/build/distributions` -- published distributions
* `allure-commandline/build/install/allure-commandline/` -- unpacked distribution so you can test it locally

**Reply by xixiaby** [reply]
Thank you. The problem is solved
